apps/src/blockTooltips/DropletFunctionTooltip.html.ejs

```diff
@@ -1,13 +1,15 @@
 <div class="function-name">
   <% if (signatureOverride) { -%>
     <%= signatureOverride %>
+  <% } else if (isProperty) { -%>
+    <% if (tipPrefix) { %><%= tipPrefix %><% } %><%= functionName %>
```

Comment:
Nit: I wonder if this concatenation should be extracted into some sort of helper since a) it happens again in a few lines, and b) it requires some special knowledge about the contents of `tipPrefix` (namely, that it includes the joining . operator).